tests/contest/runtimetest/src/tests.rs

```diff
@@ -376,3 +380,117 @@ pub fn validate_scheduler_policy(spec: &Spec) {
         eprintln!("error due to sched_nice want {want_sn}, got {sn}")
     }
 }
+
+pub fn test_io_priority_class_rt(spec: &Spec) {
+    let io_p = spec
+        .process()
+        .as_ref()
+        .unwrap()
+        .io_priority()
+        .as_ref()
+        .unwrap();
+    if io_p.class() != IoprioClassRt {
+        let io_class = io_p.class();
+        return eprintln!("error io_priority class want {IoprioClassRt:?}, got {io_class:?}");
+    }
+    // 1 is set test priority
+    if io_p.priority() != 1 {
+        let priority = io_p.priority();
+        return eprintln!("error io_priority priority want 1, got {priority}");
+    }
+
+    let ioprio_who_progress: libc::c_int = 1;
+    let ioprio_who_pid = 0;
+    let res = unsafe { libc::syscall(libc::SYS_ioprio_get, ioprio_who_progress, ioprio_who_pid) };
+    if let Err(e) = Errno::result(res) {
+        return eprintln!("error ioprio_get error {e}");
+    }
+
+    let class = res as u16 >> 13;
+    let priority = res as u16 & 0x1FFF;
+    // 1 is IoprioClassRt
+    if class != 1 {
+        return eprintln!("error ioprio_get class want {IoprioClassRt:?}, got {class}");
+    }
+
+    // because test case  priority set value is 1
+    if priority != 1 {
+        eprintln!("error ioprio_get priority want 1, got {priority}")
+    }
+}
+
+pub fn test_io_priority_class_be(spec: &Spec) {
+    let io_p = spec
+        .process()
+        .as_ref()
+        .unwrap()
+        .io_priority()
+        .as_ref()
+        .unwrap();
+    if io_p.class() != IoprioClassBe {
+        let io_class = io_p.class();
+        return eprintln!("error io_priority class want {IoprioClassRt:?}, got {io_class:?}");
+    }
+    // 1 is set test priority
+    if io_p.priority() != 1 {
+        let priority = io_p.priority();
+        return eprintln!("error io_priority priority want 1, got {priority}");
+    }
+
+    let ioprio_who_progress: libc::c_int = 1;
+    let ioprio_who_pid = 0;
+    let res = unsafe { libc::syscall(libc::SYS_ioprio_get, ioprio_who_progress, ioprio_who_pid) };
+    if let Err(e) = Errno::result(res) {
+        return eprintln!("error ioprio_get error {e}");
+    }
+
+    let class = res as u16 >> 13;
+    let priority = res as u16 & 0x1FFF;
```

Comment:
May I ask you to comment on these magic numbers?

Reply:
https://docs.kernel.org/block/ioprio.html